examples/with-react-native-web/packages/ui/src/button.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+import {
+  Pressable,
+  StyleSheet,
+  GestureResponderEvent,
+  Text,
+} from "react-native";
+
+export interface ButtonProps {
+  text: string;
+  onClick?: ((event: GestureResponderEvent) => void) | null | undefined;
```

Comment:
**thought:** I don't think the explicit `undefined‏` is needed here since we already specify the property as being optional with `?`

Reply:
Yep you're right! I don't really need the null here either.